corehq/apps/receiverwrapper/views.py

```diff
@@ -97,12 +104,18 @@ def form_ok(form_json):
         return False
 
     def case_block_ok(case_updates):
+        """
+        Check for all cases that we are submitting as demo_user and that the domain we
+        are submitting against for any previously existing cases matches the submission
+        domain.
+        """
         case_ids = set()
         for case_update in case_updates:
             case_ids.add(case_update.id)
             create_action = case_update.get_create_action()
             update_action = case_update.get_update_action()
             index_action = case_update.get_index_action()
+            # todo: Should these all (or at least the owner-based ones) also check demo_user_group_id?
```

Comment:
Yeah interesting. I wonder how often that's used? Any new domain that hasn't turned off secure submissions would have bumped into this